[Fundamentals] Migrating 'Updating billing info' from KB (#4914)
cloudflare/cloudflare-docs: content/fundamentals/get-started/basic-tasks/account-maintenance/updating-billing-info.md

---
pcx-content-type: how-to
title: Update billing information
weight: 2
---

# Update billing information

To avoid potential disruptions in your Cloudflare services, make sure your billing information is current and accurate. It is also important to understand [how Cloudflare plans and add-ons are priced](https://www.cloudflare.com/plans/#overview).

> To access your current payment methods (primary and backup) in the [Cloudflare dashboard](https://dash.cloudflare.com), navigate to the left sidebar and select **Manage account** > **Billing**. Then select **Payment Info**.

Reviewer: I might move this down into its own section

If Cloudflare is unable to process your payment, review [Troubleshooting failed payments](https://support.cloudflare.com/hc/en-us/articles/218344877).

---

## Add or update payment methods

A primary payment method is required to process payment for Cloudflare products and services. We value your confidentiality and privacy - Cloudflare does not have access to your bank, credit card, pin, or PayPal account password details.

{{<Aside type="note">}}

Customers in the Enterprise plan have additional payment options such as wire and ACH as well as yearly or quarterly billing. To discuss upgrading to Enterprise, [contact us](https://www.cloudflare.com/plans/enterprise/contact/).

{{</Aside>}}

To add or update your primary payment method,

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com).

2. On the left sidebar, select **Manage account** > **Billing**.

> 3. Select **Payment Info***.

Reviewer: ```suggestion
3. Select **Payment Info**.
```

4. If there is no payment method on file, the **Payment Method** dialog opens automatically. Otherwise, select **Manage** next to your current payment method and the **Payment Method** dialog opens.

5. Enter the required information based on the payment options below:
    * If paying by credit card:
        * Enter all the information required under the **Credit Card Details** form.

>         * To add your business' information to your invoice, include your **Company** and **VAT/GST Number**, if applicable.

Reviewer: ```suggestion
        * To add your business information to your invoice, include your **Company** and **VAT/GST Number**, if applicable.
```

{{</Aside>}}

6. Review the payment method and contact information to ensure it is accurate.

7. To finish, select **Confirm**.

Ensure your new payment method now appears in the **Payment Method** section.

## Add a backup payment method

A backup payment method is used if the primary payment method fails. To add a backup payment method:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com).

2. On the left sidebar, select **Manage account** > **Billing**.

3. Select **Payment Info**.

4. In the **Payment Method** card, select **Manage**.

5. In the **Backup** card, select **Add** to enter a backup payment method.

6. Enter the required information based on your preferred payment method (credit card or PayPal) and select **Confirm**.

7. If you would like to make the backup payment method the primary method, select **Make primary payment method** in the **Backup** card.

## Delete your current payment method

Before removing your payment method from file, you must cancel all Cloudflare paid services.

{{<Aside type="warning">}}

> If you currently subscribe to any add-on services, Cloudflare must always have a payment method on file. If you need to remove a payment method, then you must enter a new one to replace it. Otherwise, you will see an error.

Reviewer: ```suggestion
If you currently subscribe to any [add-on services](https://support.cloudflare.com/hc/articles/115004555148), Cloudflare must always have a payment method on file. If you need to remove a payment method, then you must enter a new one to replace it. Otherwise, you will see an error.
```

> ---

Reviewer: You also need to add the folder for account maintenance and then a `_index.md` file w/in that folder.

For an example, refer to https://github.com/cloudflare/cloudflare-docs/blob/production/content/bots/bot-analytics/_index.md?plain=1

pcx-content-type: how-to

> title: Updating billing information

Reviewer: ```suggestion
title: Update billing information
```

Reviewer: Might have given you bad instructions, but we try to avoid gerunds due to non-native speakers.

weight: 2
---

> # Overview

Reviewer: This should be the same title as the one specified in the frontmatter above. Not super ideal that we don't have this automated... but.

> To avoid potential disruptions in your Cloudflare services, it is important to ensure your billing information is current and accurate. It is also important to understand [how Cloudflare plans and add-ons are priced](https://www.cloudflare.com/plans/#overview).

Reviewer: ```suggestion
To avoid potential disruptions in your Cloudflare services, make sure your billing information is current and accurate. It is also important to understand [how Cloudflare plans and add-ons are priced](https://www.cloudflare.com/plans/#overview).
```

7. If you would like to make the backup payment method the primary method, select **Make primary payment method** in the **Backup** card.

## Delete your current payment method

Before removing your payment method from file, you must cancel all Cloudflare paid services.

{{<Aside type="warning">}}

If you currently subscribe to any add-on services, Cloudflare must always have a payment method on file. If you need to remove a payment method, then you must enter a new one to replace it. Otherwise, you will see an error.

You also cannot delete your payment method if a payment fails or if there is an outstanding balance. Until we process payment, you can only add or edit your payment method.

{{</Aside>}}

To delete your current payment method:

1. Log in to the Cloudfare dashboard.

2. Select **Billing**.

3. On the left navigation, select **Payment Info***.

4. Next to your current payment method, select **Delete**.

5. Select **Confirm** to finish.

Now, the **Payment Method** dialog should appear in the **Payment Method** section, where you can enter new payment details.

## Update your billing address

> The billing address is associated with your **Payment method** and is used to calculate your sales tax. If you need to update your billing address, you must also enter you **Payment Method**. The process for updating your billing address depends on the **Payment Method**.

Reviewer: Capitalization consistency with **Payment Method/method**?

If paying by credit card:

> 1. Log in to the Cloudflare dashboard.

Reviewer: ```suggestion
1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com).
```

Reviewer: Should apply this to wherever you're mentioning logins (gets folks a bit closer to their destination).

> 2. Select **My Profile** > **Billling**.

Reviewer: These steps are actually inaccurate. 

Login in > choose account > manage account > billing is the actual flow.

I've built up a lot of scar tissue around this over time and now -- when I see step-by-step instructions w/in the KB -- I always double check them myself by going through the UI. Folks are generally not updating these consistently to align with UI changes.
